### DISCUSSION TITLE
Would there be an Android release?

### DISCUSSION THREAD

**Discussion by Quarlify** [TagStudioDev/TagStudio#316] [Q&A]
I wonder if Tag Studio will be ever released on Android or Apple

**Comment by CyanVoxel** [comment]
An Android release may be possible far down the line, but it's unlikely that there would be an iOS release due to Apple's filesystem and sandboxing restrictions on the platform. There could possibly be a version that purely views files you have self-hosted on another machine instead of the files directly on your iPhone, but again that would be a long time from now if it ever happened.

**Comment by DrRetro2033** [comment]
Its possible with a Flutter front-end or some other cross platform framework, however, we first need to improve the back-end a lot more before we do anything too drastic.
